### DISCUSSION TITLE
Import product feeds into internal store taxonomy

### DISCUSSION THREAD

**Discussion by simplexityware** [vendurehq/vendure#1748] [Q&A]
How does the tool accomplish the above?

Imagine developing an aggregator e-commerce site that attempts to map multiple product feeds into one consistent taxonomy in one store to provide one stop convenient  shopping experience.  Each product feed may specify its products differently from another.  This is actually a natural language processing problem.  Does vendure offer any assistance in this area?

**Comment by michaelbromley** [comment]
Hi,

No, Vendure does not have anything specifically to handle this topic. As you point out, the main work would be processing the various inputs into a consistent and standardized output taxonomy, which is outside the domain of what Vendure seeks to solve.

Once that part is solved, importing that data into Vendure is certainly possible - that aspect is covered here: https://www.vendure.io/docs/developer-guide/importing-product-data/
